src/client/app/offers/offers.js

```diff
@@ -1,19 +1,39 @@
-﻿(function () {
+(function () {
     'use strict';
 
     angular
         .module('app.offers')
         .controller('OffersController', OffersController);
 
-    OffersController.$inject = ['$location'];
+    OffersController.$inject = ['$scope', '$location'];
 
-    function OffersController($location) {
+    function OffersController($scope, $location) {
```

Comment:
do not inject `$scope`.
use `vm` instead
it binds to scope via `controllerAs` property in a router